modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -108,6 +108,10 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |The `registry.access.redhat.com` and `https://registry.redhat.io` sites redirect through `catalog.redhat.com`.
 
+|`*.cloudfront.net`
+|443
+|Allows for CloudFront support.
```

Comment:
change to:
"Used by ROSA for its STS implementation with managed OIDC configuration."
or similar.